react/components/SearchBar/components/SearchBar.js

```diff
@@ -186,7 +186,7 @@ const SearchBar = ({
             <div
               className={classNames(
                 'relative-m w-100',
-                handles.searchBarInnerContainer
+                applyModifiers(handles.searchBarInnerContainer, [isOpen ? 'opened' : '', inputValue ? 'filled' : '']),
```

Comment:
There are some lint problems, you can fix this by installing the dependencies from the root of the app and running `yarn lint --fix`